homeassistant/components/unifi/button.py

```diff
@@ -42,6 +48,15 @@ async def async_restart_device_control_fn(
     await api.request(DeviceRestartRequest.create(obj_id))
 
 
+@callback
```

Comment:
A coroutine function isn't a callback. We can remove the decorator.

Reply:
https://github.com/home-assistant/core/pull/112276